stdlib/public/core/Arrays.swift.gyb

```diff
@@ -2381,6 +2381,11 @@ extension ${Self} {
 %end
 
 #if _runtime(_ObjC)
+@_versioned internal
```

Comment:
Please add a comment describing why this is here. Otherwise, someone will just delete it